### ISSUE TITLE
Question: are social/gateway/other tokens usable for NFTs or Bounties?

### ISSUE THREAD

**Issue by mochet** [Joystream/joystream#3332]
## Question: are social/gateway/other tokens usable for various purposes?
1. If I have creator or gateway tokens, can I use them to launch an NFT sale?
2. If I have creator or gateway tokens, can I use them to bid on an NFT sale?
3. If I have creator or gateway tokens, can I use them to fund a bounty?
4. Is there any plan to introduce any of the first 3 prior to mainnet launch? (likely not, but just want to be aware)
5. Is there anywhere specific these can/cannot be used? Or is the current situation basically "they can't be used in NFTs or bounties whatsoever"?

## Feedback
* If this hasn't been considered before, then it may provide a lot of value in allowing NFTs and bounties to be supported, with creation fees having to be JOY tokens but actual funding allowing other token types.
* Generally, anything that provides more ways to use secondary tokens on the platform will give them a lot more potential usage, and have ways for the platform to still capture value via `fees` or other methods.
* It could maybe be an option to allow/restrict non-JOY tokens if this at all sound feasible.
_(out of scope: nothing core-function related like council, proposals, forum or video comments)_

**Comment by bedeho** [comment]
> If I have creator or gateway tokens, can I use them to launch an NFT sale?

Any creator can sell video NFTs, being able to do this is not contingent on creator tokens. Gateway tokens are so far from being a thing I can't even say anything sensible.

> If I have creator or gateway tokens, can I use them to bid on an NFT sale?

No, only $JOY, it is the base currency for everything.

> If I have creator or gateway tokens, can I use them to fund a bounty?

No, same.

> Is there any plan to introduce any of the first 3 prior to mainnet launch? (likely not, but just want to be aware)

Nope ;)

> Is there anywhere specific these can/cannot be used? Or is the current situation basically "they can't be used in NFTs or bounties whatsoever"?

Creator tokens can only be used, on-chain, to claim revenue shares from a channel. We can very easily give other powers, but that is the starting point.

**Comment by mochet** [comment]
> Gateway tokens are so far from being a thing I can't even say anything sensible.
* I really think there is a potentially significant, unrealized proposition in this. I will share something soon covering this.

> Creator tokens can only be used, on-chain, to claim revenue shares from a channel. We can very easily give other powers, but that is the starting point.
* Understandable and despite changes being a solid `Nope ;)`.... would still be cool to see channels participating in funding platform improvements + projects. It could also potentially make a two fold investment/value proposition for people doing bounty work as they would also be gaining tokens for channels as an additional reward outside of pure `JOY` incentives.
